Sources/Basic/OutputByteStream.swift

```diff
@@ -48,62 +50,94 @@ public class OutputByteStream: OutputStream {
         self.buffer = []
```

Comment:
We should probably create the buffer with the expected capacity, so it never needs to grow. I forget the exact initializer for that.